recipes/reseek/vcxproj_make.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
```

Comment:
where is this coming from? Maybe add a comment here, that this file needs to be kept in sync?

Reply:
This is the author's own script. In his repository he downloads it at build time, but I decided to have it within the recipe to have more control over it. I'll add a comment.